easybuild/easyconfigs/t/TOML-Fortran/TOML-Fortran-0.2.2-GCC-10.2.0.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'CMakeMake'
+
+name = 'TOML-Fortran'
+version = '0.2.2'
+
+homepage = 'https://github.com/toml-f/toml-f'
+description = 'TOML parser for Fortran projects'
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/toml-f/toml-f/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['c3e5e1f57de00977b5e41d3a857ddb7e6b41f0b9116ed608cf89f295d9ef24c6']
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+]
+
+separate_build_dir = True
```

Comment:
This is already done by default by the modern `CMakeMake` easyblock (see `eb -a -e CMakeMake | grep separate_build_dir`), so can be removed...

```suggestion
```